### ISSUE TITLE
Replay Options Text display is slightly messed up [work around solution found] 

### ISSUE THREAD

**Issue by NicKnack819** [ReplayMod/ReplayMod#101]
The text options aren't displaying properly
![2020-02-01_15 43 34](https://user-images.githubusercontent.com/60555295/73598889-849c1600-450b-11ea-956b-176a3de036f2.png)
![2020-02-01_15 57 55](https://user-images.githubusercontent.com/60555295/73598907-a7c6c580-450b-11ea-91f5-ba36fe9c9f4e.png)
![2020-02-01_15 58 03](https://user-images.githubusercontent.com/60555295/73598908-ac8b7980-450b-11ea-91fa-2b9e98469383.png)

A Solution was found 
> That fixed it
> use the one below for future reference to anyone who reads this
> https://www.curseforge.com/minecraft/mc-mods/fabric-api/download/2810785

**Comment by haykam821** [comment]
What version of [Fabric API](https://www.curseforge.com/minecraft/mc-mods/fabric-api) do you have installed?

**Comment by NicKnack819** [comment]
> What version of Fabric API do you have installed?

The most recent one for 1.14.4:

fabric-api-0.4.2+build.246-1.14.jar

**Comment by NicKnack819** [comment]
What does that mean @Johni0702 ?

**Comment by Johni0702** [comment]
What does what mean?
No need to apologize for the ping, I'm subscribed to all issues in this repo anyway, so pings merely add clarity as to whom you're talking to.

Wrt the issue, could you try an older fabric-api version (0.3.x)? The ReplayMod hooks into a specific internal thing of fabric-api and that thing changing would explain the broken text.

**Comment by SerenModz21** [comment]
i switched my fabric api to `fabric-api-0.3.3+build.234-1.14` and the text has been fixed for me

**Comment by NicKnack819** [comment]
That fixed it
use the one below for future reference to anyone who reads this
https://www.curseforge.com/minecraft/mc-mods/fabric-api/download/2810785

**Comment by NicKnack819** [comment]
do I comment and close now @Johni0702 ?

**Comment by Johni0702** [comment]
@NicKnack819 We'll keep it it open until it works with the new version as well.

**Comment by NicKnack819** [comment]
Ok @Johni0702 . Is the a format I should add to show the current solution?

**Comment by Johni0702** [comment]
You can edit it to the end of the top post but given how short this discussion was, I expect anyone which finds this issue in search of a workaround to also be able to skim over 5 replies to find it.

**Comment by NicKnack819** [comment]
@Johni0702 good?

**Comment by drewc5131** [comment]
An easy fix was to just copy the language data from the mod into the en_us.json of actual minecraft, then make a resource pack for it to just overwrite it, it allows you to use the latest version of fabric api while still getting the language. Put it into your resource packs folder and apply it.
[ReplayLangFix.zip](https://github.com/ReplayMod/ReplayMod/files/4271333/ReplayLangFix.zip)

**Comment by NicKnack819** [comment]
> An easy fix was to just copy the language data from the mod into the en_us.json of actual minecraft, then make a resource pack for it to just overwrite it, it allows you to use the latest version of fabric api while still getting the language. Put it into your resource packs folder and apply it.
> [ReplayLangFix.zip](https://github.com/ReplayMod/ReplayMod/files/4271333/ReplayLangFix.zip)
@drewc5131 
Could you try to put that in layman's terms please?

**Comment by drewc5131** [comment]
> 
> 
> > An easy fix was to just copy the language data from the mod into the en_us.json of actual minecraft, then make a resource pack for it to just overwrite it, it allows you to use the latest version of fabric api while still getting the language. Put it into your resource packs folder and apply it.
> > [ReplayLangFix.zip](https://github.com/ReplayMod/ReplayMod/files/4271333/ReplayLangFix.zip)
> 
> @drewc5131
> Could you try to put that in layman's terms please?

new minecraft uses new language format
mod does not use new format

you can easily see what ive done if u just look at the en_US.json file and scroll to the bottom
